ios/FluentUI/Vnext/Drawer/Drawer.swift

```diff
@@ -95,6 +91,9 @@ public struct MSFDrawerView<Content: View>: View {
                 .performOnBackgroundTap {
                     state.isExpanded = false
                 }
+                .transitionCompletion {
+                    state.didChangeState?(state.isExpanded)
```

Comment:
It seems like the first time the drawer is presented, the animation is not happening correctly:


https://user-images.githubusercontent.com/68076145/106037127-1ab93c80-608b-11eb-8444-4c6dbadb7ce5.mov

Reply:
Thanks for catching this, missed the bug as this does not replicate on iOS 13. I wish push changes to fix this problem.